src/components/Input.tsx

```diff
@@ -6,42 +6,62 @@ import { AriaLiveErrorMessage } from './AriaLiveErrorMessage';
 type InputProps = {
   register: any;
   type?: string;
-  name: 'name' | 'email';
+  name: string;
+  label: string;
   placeholder?: string;
   errors: FieldErrors;
-  watchedValue?: string;
+  watchedValue?: string | number | Date;
   className?: string;
+  isEventInput?: boolean;
 };
 
 export const Input: React.FC<InputProps> = ({
   register,
   type = 'text',
   name,
+  label,
   placeholder,
   errors,
   watchedValue,
   className = '',
+  isEventInput = false,
```

Comment:
Let's avoid coupling when it is this simple. Generic input component or any generic component of any sort must not know about the placement. 

You will add a new form for activities and it will have `isActivity`? Then `isActivityInTheBottom`. `isLogin`. After a while, you are in an absolute apropcalypse :D 

Then you will end up with this. That is btw GENERIC button

![CleanShot 2023-11-14 at 16 24 38@2x](https://github.com/GLObus303/sportujspolu/assets/12488648/4e693f32-7ceb-493e-a1a9-fb70245215f2)

Reply:
Solution: expose classnames: `labelClassName` f.e. and similar, feel free to create a level of abstraction if necessary - f.e EventInput. 

I am not sure what you are changing, but it feels like something that should not be present here at all. 
a) input should look the same across the app
b) margin bottom should either universal or handled in the place of usage.